web/src/components/molecules/Settings/Project/Dataset/DatasetList/index.stories.tsx

```diff
@@ -14,4 +14,10 @@ export default {
   component: DatasetList,
 } as Meta;
 
-export const Default = () => <DatasetList items={items} />;
+export const Default = () => (
+  <DatasetList
+    items={items}
+    removeDatasetSchema={schemaId => console.log(`Removing dataset with ID ${schemaId}`)}
+    onDownloadFile={(id, name) => console.log(`Downloading file with ID ${id} and name ${name}`)}
```

Comment:
I think you should use `@storybook/addon-actions` to mock action like the below.
```js
removeDatasetSchema={action("removeDatasetSchema")}
onDownloadFile={action("onDownloadFile")}
```

Reply:
Sure , Done